pkg/instrumentation/defaultinstrumentation_test.go

```diff
@@ -132,12 +167,12 @@ func Test_getDefaultInstrumentation(t *testing.T) {
 		wantErr bool
 	}{
 		{
-			name: "appsignals-http",
+			name: "application-signals-http",
 			args: args{
 				agentConfig: &adapters.CwaConfig{
 					Logs: &adapters.Logs{
 						LogMetricsCollected: &adapters.LogMetricsCollected{
-							AppSignals: &adapters.AppSignals{},
```

Comment:
AppSignals is not used anymore, it's safe to remove. I can have a follow-up PR to remove